python/mxnet/symbol.py

```diff
@@ -1025,6 +1025,36 @@ def grad(self, wrt):
         return Symbol(handle)
     # pylint: enable= no-member
 
+    def eval(self, args=None, ctx=cpu()):
+        if args == None:
```

Comment:
Oh I meant removing the line `print("hello world!")` and move these two lines down to under the docstring